src/features/defi/providers/fox-farming/components/FoxFarmingManager/Claim/ClaimStatus.tsx

```diff
@@ -156,6 +188,53 @@ export const ClaimStatus: React.FC<ClaimStatusProps> = ({ accountId }) => {
 
   const handleClose = useCallback(() => browserHistory.goBack(), [browserHistory])
 
+  const txLink = useMemo(() => {
+    if (!feeAsset) return
+    if (!txid) return
+
+    if (maybeSafeTx?.transaction?.transactionHash)
+      return getTxLink({
+        txId: maybeSafeTx.transaction.transactionHash,
+        defaultExplorerBaseUrl: feeAsset.explorerTxLink,
+        accountId,
+        // on-chain Tx
```

Comment:
So.. thinking about this some more, this seems like a lot of flaky and duplicated logic - if *isSafeTxHash* we should pass both a `safeTxHash` (defined) and `txid` (maybe undefined)